.gitpod.Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM gitpod/workspace-full:latest
+
+USER gitpod
+
+# Install .NET SDK
+ADD global.json ./
+RUN mkdir -p /home/gitpod/dotnet \
+    && curl -fsSL https://dot.net/v1/dotnet-install.sh | bash /dev/stdin --jsonfile global.json --install-dir /home/gitpod/dotnet \
+    && rm global.json
+ENV DOTNET_ROOT=/home/gitpod/dotnet
+ENV PATH=$PATH:/home/gitpod/dotnet
+
+# Install Consul Server
+ADD Directory.Build.props ./
+RUN CONSUL_VERSION=$(cat Directory.Build.props|grep -oP '(?<VersionPrefix>)\d{1,3}\.\d{1,3}\.\d{1,3}') \
```

Comment:
hmmm I'm not sure about this. I think you are mixing the consuldotnet version with the consul (server) version?

I guess any of these could be used instead https://github.com/G-Research/consuldotnet/blob/da1995acd819e52fb16ba2b9375dc601cb91e300/.github/workflows/ci.yml#L60

Reply:
Actually, I take the first 3 parts of consuldotnet version (according to the [versioning section in README.md](https://github.com/G-Research/consuldotnet#versioning)).
Let me know if I should use a different version.

Reply:
I think that as long as the version of consuldotnet is correlated with version of consul server, this is a correct way to get the version of consul server.